index.js

```diff
@@ -120,9 +120,15 @@ async function installCrystalForLinux({crystal, shards, arch = getArch(), path})
     const filePatterns = {"x86_64": /-linux-x86_64\.tar\.gz$/, "x86": /-linux-i686\.tar\.gz$/};
     checkArch(arch, Object.keys(filePatterns));
 
-    const depsTask = installAptPackages(
-        "libevent-dev libgmp-dev libpcre2-dev libssl-dev libxml2-dev libyaml-dev".split(" "),
-    );
+    libs = "libevent-dev libgmp-dev libssl-dev libxml2-dev libyaml-dev".split(" ");
+    if (crystal === Latest || crystal === Nightly || cmpTags(crystal, "1.2") >= 0) {
+        libs.push("libpcre2-dev");
```

Comment:
Eh, that would install *only* PCRE2 for Crystal 1.2 and above? But PCRE2 support is only fully available in 1.8.

I guess the version comparison should be with `1.8`?
`Latest` is a moving target. Right now it's 1.7.3 and we need libpcre. After the release of 1.8 it's 1.8.0 and we need libpcre2. So we must either install both for `Latest`, or figure out whether `Latest` means 1.8+. I suppose this could be done by installing crystal first and then checking which version was installed.

However, there's no harm in installing both libraries and I'd suggest to go with that at least for `Latest`, and probably also for 1.8.